lzero/model/common.py

```diff
@@ -35,9 +49,121 @@ class MZNetworkOutput:
     latent_state: torch.Tensor
 
 
+class SimNorm(nn.Module):
+
+    def __init__(self, simnorm_dim):
+        """
+        Overview:
+            Simplicial normalization. Adapted from https://arxiv.org/abs/2204.00616.
+        Arguments:
+            - simnorm_dim (:obj:`int`): The dimension for simplicial normalization.
+        """
+        super().__init__()
+        self.dim = simnorm_dim
+
+    def forward(self, x):
+        """
+        Overview:
+            Forward pass of the SimNorm layer.
+        Arguments:
+            - x (:obj:`torch.Tensor`): The input tensor to normalize.
+        Returns:
+            - :obj:`torch.Tensor`: The normalized tensor.
+        """
+        shp = x.shape
+        # Ensure that there is at least one simplex to normalize across.
+        if shp[1] != 0:
+            x = x.view(*shp[:-1], -1, self.dim)
```

Comment:
replace "view" to "reshape", which is safer

Reply:
but "view" is more efficient here?

Reply:
view is more siutable here